### ISSUE TITLE
Responsive, accessibility issue: Content too wide

### ISSUE THREAD

**Issue by rfultz** [fecgov/fec-cms#3860]
Google Search Console is warning us that we have some pages with content wider than mobile screens (and larger issues).

The pages:
- https://transition.fec.gov/law/litigation_CCA_Alpha.shtml **This is a big project because we haven't brought over court case pages**
- https://www.fec.gov/updates/pac-activity-increases-in-2000-election-cycle/ **This one has been updated**
- https://www.fec.gov/updates/mandatory-electronic-filing-rules-published/ **This one has been updated**
- https://www.fec.gov/updates/lawrence-h-norton-named-general-counsel/ **This one has been updated** 
- https://www.fec.gov/updates/fec-reports-on-congressional-financial-activity-for-2000/
- https://www.fec.gov/law/litigation/rtao.shtml **A court case page.**
- https://www.fec.gov/updates/pac-activity-continues-climb-in-2006/  **This one has been updated** 
- https://www.fec.gov/updates/hearing-set-for-march-24-on-proposed-public-funding-rule/ **This one has been updated** 
- https://transition.fec.gov/rad/ **The RAD FAQs are being brought into Wagtail via Brad's RAD tickets; we'll redirect then**
- https://transition.fec.gov/law/legalconsideration.shtml **We're transitioning this to Wagtail in this PI and we'll redirec then**
- https://www.fec.gov/fosers/ **This is a searchable legal system that will be part of IT modernization plans**
- https://transition.fec.gov/pages/report_notices/2018/pa18.shtml - **This will require a master in HTML. I think it is one for the front end team.** 
- https://www.fec.gov/updates/fec-clarifies-policy-in-subpoena-enforcement-issues-declines-to-seek-further-court-review-on-specific-case/ **This one has been updated** 
- https://transition.fec.gov/pages/report_notices/2017/mt00.shtml **-This will require a master in HTML. I think it is one for the front end team.**
- https://transition.fec.gov/pubrec/stateelection.shtml **Public Records is migrating this page to Wagtail; we'll redirect then**
- https://transition.fec.gov/pages/report_notices/2017/ut03.shtml - **This will require a master in HTML. I think it is one for the front end team.**

### Completion criteria
- [x] Pages above that are scraped press releases (with /updates/) are cleaned up.
- [x] Plan made to address report notices (move to Wagtail?, upload PDF?) and redirects
- [x] Plan made to address court case pages
- [ ] Plan made about whether to have contractor address SERS page.
- [x] Tickets put in for future work

**Comment by JonellaCulmer** [comment]
This may be good overall ticket to audit these pages and others that don't fit into our current style. 

Which of these pages can be updated/improved? And how?

Like this one: https://www.fec.gov/regulations/about, which was not on your original list but could do with some improvements.

**Comment by dorothyyeager** [comment]
Issues with the press releases can be resolved in this ticket: #3655 

We have larger decisions to make about the transition pages.

**Comment by dorothyyeager** [comment]
I put in a ticket to resolve the redirect issue @kathycarothers is running into with some of the press releases.

**Comment by kathycarothers** [comment]
I have to put this one on hold until we get the redirects fixed for some of the links on this page. It is drafted.

**Comment by dorothyyeager** [comment]
@kathycarothers you may want to work on the other press releases to see if there are similar redirect issues going on with them.

>   It is drafted.

Wasn't sure which one of the pages above you meant (I know it's a press release).

**Comment by kathycarothers** [comment]
The release is blocked as we need to fix the redirect issues with this 2000 release https://www.fec.gov/updates/pac-activity-increases-in-2000-election-cycle/  @dorothyyeager

**Comment by kathycarothers** [comment]
This ticket is blocked based on redirects and moving pages off of transition. Please see the notes above. This ticket is blocked by #3885 and #3816. Will need to have another discussion on these pages in the future.

**Comment by kathycarothers** [comment]
Transition pages involving the prior notices noted above will be handled in ticket #3935 
Transition page for ongoing legal consideration will be handled in ticket #3937 and is referenced in ticket #3470 
Transition page for court case alpha will be handled in ticket #3938 
The Public Records pages and redirects will be handled in ticket #3936 
The RAD FAQs are being handled in ticket #3350 
The Press Releases redirects will be handled in ticket #3835 (the developments of that meeting will determine what happens with the links in these releases that are not working.)

**Comment by JonellaCulmer** [comment]
The only other page on this list that is not addressed anywhere else is the SERS search page. The migration of this search is on a longer to-do list. So for now, we will close this issue, as the problem has been mentioned to the team handling SERS.
